generators/app/templates/package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "<%=projectName%>",
+  "version": "0.1.0",
+  "description": "<%=projectName%>",
+  "main": "lib/<%=projectClassName%>.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/<%=projectName%>.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "<%=projectName%>",
+    "<%=projectClassName%>",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/<%=projectName%>/issues"
+  },
+  "homepage": "https://github.com/cerner/<%=projectName%>#readme",
+  "scripts": {
+    "build": "babel src --out-dir lib",
+    "lint": "npm run lint-js && npm run lint-scss",
+    "lint-js": "eslint --ext .jsx --ext .js .",
+    "lint-scss": "stylelint src/**/*.scss",
+    "posttest": "npm run lint",
+    "pretest": "npm run build", 
+    "prepublish": "npm run test",
+    "release:major": "npm version major -m \"Released version %s\" && npm publish && git push --follow-tags",
+    "release:minor": "npm version minor -m \"Released version %s\" && npm publish && git push --follow-tags",
+    "release:patch": "npm version patch -m \"Released version %s\" && npm publish && git push --follow-tags",
+    "start": "npm run lint && webpack-dev-server --hot --inline --display-error-details --progress --colors --host 0.0.0.0",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:coverage": "jest --coverage"
+  },
+  "jest": {
+    "coveragePathIgnorePatterns": ["/node_modules/", "/tests"],
+    "moduleNameMapper": {
+      "\\.(css|less|scss)$": "identity-obj-proxy"
+    },
+    "setupFiles": ["./tests/jestsetup.js"],
```

Comment:
I'm including this in case a developer might need it but currently it's blank.  Is it worth just dropping this from the generator?